lib/error-handler.js

```diff
@@ -12,10 +26,16 @@ module.exports = {
     }).install();
 
     app.use(Raven.requestHandler());
-    app.use(timeout('10s'));
+
+    const timeoutValue = '10s';
+    app.use(timeout(timeoutValue));
```

Comment:
The [docs](https://www.npmjs.com/package/connect-timeout) for `connect-timeout` add this caveat:
 
> While the library will emit a 'timeout' event when requests exceed the given timeout, node will continue processing the slow request until it terminates.

Is it okay that this won't stop processing the request?

Reply:
For now, I don't want to alter the behavior. 
The `next` middleware is actually adding an on-timeout handler.
For now I only want to get more insights, but eventually, I will start aborting the process.